rotkehlchen/assets/asset.py

```diff
@@ -739,6 +739,18 @@ def to_dict(self) -> Dict[str, Any]:
     def serialize(self) -> str:
         return self.identifier
 
+    def check_existence(self) -> 'Asset':
+        """
+        If this asset exists, returns nothing. If it doesn't, throws an error.
+        May raise:
+        - UnknownAsset
+        """
+        # We don't need asset type, but using `get_asset_type` since it has all the functionality
+        # that we need here (reads only from `assets` table and raises UnknownAsset if identifier
+        # doesn't exist).
+        AssetResolver().get_asset_type(self.identifier)
+        return self  # return self just to simplify code in some places
```

Comment:
no need to specify so much in the comments. Write stuff like that if needed in the docstring